desktop-exporter/app/components/traceList.tsx

```diff
@@ -13,13 +20,38 @@ type RowProps = {
 };
 
 function Row({ index, style, data }: RowProps) {
+  let isSelected = useMatch(`traces/${data[index].traceID}`);
+
+  let backgroundColour = isSelected
+    ? useColorModeValue("teal.100", "teal.700")
+    : useColorModeValue("whiteAlpha.700", "whiteAlpha.400");
+
   return (
-    <NavLink
-      to={`traces/${data[index].traceID}`}
-      style={style}
-    >
-      {data[index].traceID}
-    </NavLink>
+    <div style={style}>
+      <LinkBox
+        bgColor={backgroundColour}
+        height="100px"
+        padding="10px"
+        marginX="10px"
+        marginTop="10px"
+        rounded="md"
+      >
+        <Heading
+          marginY="1"
+          noOfLines={1}
+          size="sm"
+        >
+          <LinkOverlay
```

Comment:
Just to make sure, do you have to click on the trace id or can you click anywhere in the box?

Reply:
You click the LinkBox. The link overlay is the link inside the box that gets 'lifted' (not sure of the right word).